docs/changelog/4.3.rst

```diff
@@ -1,16 +1,38 @@
 Changelogs for 4.3.x
 ====================
 
+.. changelog::
+  :version: 4.3.0-beta1
+  :released: 31st of January 2020
+
+  This is the first beta for version 4.3.0 of the Authoritative Server.
+  Please see :doc:`the upgrade notes <../upgrading>` for some minor breaking changes.
+
+  Some minor fixes have been left out from the list below; some other bugs may have been fixed accidentally as a result of the tremendous amount of work that goes into each of our major releases.
+
+  As announced in `Backend removals in the upcoming Authoritative Server release <https://blog.powerdns.com/2019/10/17/backend-removals-in-the-upcoming-authoritative-server-release/>`_, we have removed five backends.
+
+8749     allow local-ipv6 until 4.4.0
+8594    auth: add default-publish-{cds|cdnskey} options
+8744     Make Lua mandatory for Auth
+8681     auth: Add metrics about the size of our in-memory rings
+8628     auth: make sure get() is always returning the default value for d_place
+8627     gpgsqlbackend: Avoid actually prepared statements
+8713     auth: Enforce a strict maximum size for the packet and records caches
+8701    remote: Support ::alsoNotifies
+8649     auth: Clear the TSIG algo between iterations in the API
+8177     Initial work for adding hidden dnssec keys
```

Comment:
FWIW, since the feature was renamed, and I think it's more or less complete, it would be more clear if 8177 was described in the changelog with something like "Add unpublished DNSSEC keys"